kong/runloop/wasm.lua

```diff
@@ -788,9 +928,10 @@ function _M.attach(ctx)
     return kong.response.error(500)
   end
 
-  set_proxy_wasm_property("kong.route_id", ctx.route and ctx.route.id)
-  set_proxy_wasm_property("kong.service_id", ctx.service and ctx.service.id)
+  proxy_wasm.set_property_getter(properties.get)
+  proxy_wasm.set_property_setter(properties.set)
```

Comment:
Note: In the final merge of this feature in ngx_wasm_module, these two APIs are merged into one: ok, err = set_host_properties_handlers(getter, setter).

Also, we should definitely catch/log the error it may return.